tests/v2/integration/setup/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/creasty/defaults"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
```

Comment:
I would name this `rancherClient` or something, so it's not confused with any "rancher" named package.

Reply:
Updated